Sources/Apodini/Apodini.docc/Testing/NegativeCompileTest.md

```diff
@@ -0,0 +1,100 @@
+# Negative Compile Test
```

Comment:
@Supereg What do you think: Should this be moved to the DocC documentation or should this stay somewhere else in the Repo for Apodini Maintainers? As far as I can see this is only usable internally and not an external feature.

Reply:
NegativeCompileTests are currently only used internally. Since it will only ever scan for tests targets in the Apodini repo at the moment, they can only be used internally.
So, at the current state, I don't think it makes sense to include them in the DocC documentation(?)